code/cynthia/Python/test_ATM_lab12.py

```diff
@@ -0,0 +1,37 @@
+#Automated Tests
+#using using ATM 
+
+from lab08_2 import ATM
+import pytest
+from unittest.mock import patch 
```

Comment:
as a general practice, the clean-up phase before submitting work (here in class, or in the real world) should include getting rid of any unused imports